roles/mysql/tasks/main.yml

```diff
@@ -37,9 +62,10 @@
 
 - name: MySQL | Restart MySQL
   ansible.builtin.service:
-    name: mysql
+    name: "{{ mysql_name }}"
```

Comment:
Double-check that we give this var a default setting in the mysql role and/or set it for all applications to avoid the `variable undefined` error.